src/main/java/com/thebois/views/ColonyView.java

```diff
@@ -0,0 +1,39 @@
+package com.thebois.views;
+
+import java.util.Collection;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+
+import com.thebois.models.beings.IBeing;
+
+/**
+ * Used to display all pawns.
+ */
+public class ColonyView {
+
+    /**
+     * Draws all pawns.
+     *
+     * @param listOfPawns list of pawns to draw.
+     * @param batch       used to draw.
+     */
+    public void drawAllPawns(ShapeRenderer batch, Collection<IBeing> listOfPawns) {
```

Comment:
This needs to be refactored to use the same setup we created in #24. 

(e.g. the view implements `IView` and it recieves the `listOfBeings` from a `BeingController`)

Reply:
Will fix this.

Reply:
You should pull down the latest master and merge it into your branch, in order to get the latest changes in MVC-structure from #24! Best to do this before implementing it yourself for `Being`s and `BeingGroup`.

Reply:
This still doesn't follow the changes we agreed upon the last meeting. The view doesn't implement IView.